src/Views/HardwareView.vala

```diff
@@ -280,6 +282,68 @@ public class About.HardwareView : Gtk.Grid {
 
         return 0;
     }
+
+    private string get_storage_type () {
+        string partition_name = get_partition_name ();
+        string disk_name = get_disk_name (partition_name);
+        string path = "/sys/block/%s/queue/rotational".printf (disk_name);
+        string storage = "";
+        try {
+            string contents;
+            FileUtils.get_contents (path, out contents);
+            if (int.parse (contents) == 0) {
+                if (disk_name.has_prefix ("nvme")) {
+                    storage = _("NVMe SSD");
+                } else if (disk_name.has_prefix ("mmc")) {
+                    storage = _("eMMC");
+                } else {
+                    storage = _("SATA SSD");
+                }
+            } else {
+                storage = _("HDD");
+            }
+        } catch (FileError e) {
+            warning (e.message);
+            storage = _("Unknown");
```

Comment:
Instead of "unknown" why not say "storage" like a fallback to the existing output?

Reply:
Thank you, addressed.